t04_refactored.py

```diff
@@ -164,10 +164,37 @@ def team_8_adv():
 
 ###################################################################################
 
-def team_9_adv():
-    pass
-    # TODO Add your code here
 
+def team_9_adv():
+    """
+    This function gives information based on whom's chosen path
+    https://docs.google.com/document/d/1P1N_fGLfx5g6j8d5jtkkOIGY4QmB8QjGBr1Yd7kFwmo/edit?usp=sharing
+    Sugarragchaa Tumur-Ochir
+    Mahmoud Leghlimi
+    :return: none
+    """
+    global direction
+    global isDead
+    if direction == "North":
+        print("""Now you have to complete three treacherous trials with the other person in the cave. 
+                Each more arduous than the last.""")
+        direction = input("""The first trial is you must befriend and then destroy a sneaky salamander. 
+        Choose yes / no , give attention these two are lowercase""")
+        if direction == "yes":
+            print("""Congratulations you can proceed to the next trial. 
+            The next trial you have to go find the other person in the cave with you and make them poke a grizzly bear. 
+            """)
+        elif direction == "no":
+            print("Due to your declination, you will now be forced to leave the labyrinth.")
+            isDead = True
+        else:
+            print(
+                """Playful stubborn little boy, I was once like you""")
+    else:
+        # neutral choice
+        print(
+            """Now you are presented with two choices. 
+            "One direction could lead to your death and the other solution is unknown""")
```

Comment:
I gave you a function to kill the user when they die. Make sure to call it here:

`kill_if_dead(isDead)`

Reply:
we called it in our nested if statement